docs-src/cloud/operating-envelope-latency.md

```diff
@@ -18,4 +18,5 @@ In June 2023, latency measurements over a week period for starting and signaling
 
 As we continue working on improving latencies, these numbers will decrease over time.
 
+Be aware that the latency observed on the Temporal Client might be impacted by other system components, such as the Codec Server, egress proxy, and the network itself.
```

Comment:
```suggestion
The latency observed from the Temporal Client might be impacted by other system components, such as the Codec Server, egress proxy, and the network itself.
```

I like avoiding commands, and to some extent every sentence in the docs is "be aware". If important enough, would add a warning block, but don't think important enough here